website/templates/report.html

```diff
@@ -383,6 +431,27 @@ <h2 class="text-2xl font-semibold leading-7 text-gray-900">{% trans "ADD TEAM ME
         return cveRegex.test(cveNumber);
     }
 
+    document.addEventListener('DOMContentLoaded', function () {
+        const labelSelect = document.getElementById("labelSelect");
+        const trademarkFields = document.getElementById("trademark-fields");
+
+        updateFormParameters(labelSelect.value);
+
+        labelSelect.addEventListener('change', function () {
+            updateFormParameters(this.value);
+        });
+
+        // Function to update form parameters based on the selected type
+        function updateFormParameters(type) {
+            if (type === "8") {
+                trademarkFields.style.display = "block";
+            } else {
+                trademarkFields.style.display = "none"; 
+            }
```

Comment:
```suggestion
            trademarkFields.style.display = (type === "8") ? "block" : "none";

```

Reply:
resolved in https://github.com/OWASP-BLT/BLT/pull/2655/commits/3e76e5b033074d878415807aa0742ac40a9e7996